grace/models/classifier.py

```diff
@@ -33,6 +33,7 @@ def __init__(
         input_channels: int,
         hidden_channels: List[int],
         *,
+        dropout: float = 0.5,
```

Comment:
Could you add this to the docstrings. Might be worth adding a sentence to the notes too, about the new node classifier